konami.js

```diff
@@ -0,0 +1,41 @@
+  var konamiCodePosition = 0;
+document.addEventListener('keydown', function(e) {
+  var allowedKeys = {
+    37: 'left',
```

Comment:
the key locations are each assigned a number and you are defining those numbers to the values that you are using in the array below.